s3_scrubber/README.md

```diff
@@ -9,11 +9,13 @@ and `safekeeper`, and does housekeeping such as cleaning up objects for tenants
 
 #### S3
 
-Do `aws sso login --profile dev` to get the SSO access to the bucket to clean, get the SSO_ACCOUNT_ID for your profile (`cat ~/.aws/config` may help).
+Do `aws sso login --profile dev` to get the SSO access to the bucket to clean.
+Also, set the following environment variables:
 
-- `SSO_ACCOUNT_ID`: Credentials id to use for accessing S3 buckets
+- `AWS_PROFILE`: Profile name to use for accessing S3 buckets (e.g. `dev`)
 - `REGION`: A region where the bucket is located at.
 - `BUCKET`: Bucket name
+- `BUCKET_PREFIX` (optional): Prefix inside the bucket
```

Comment:
I wonder what do we have this for. Probably for tests running against minio?

```suggestion
- `BUCKET_PREFIX` (optional): Prefix inside the bucket (used in regress tests)
```

Reply:
But you don't need to configure one if you launch s3 scrubber to snapshot a tenant, do you?

Reply:
I've pointed out in slack that if no prefix is specified, we use `pageserver/v1` as the prefix which is the one we also use in prod so if you hit the prod buckets you don't need to specify the prefix. Still important to document it.